shared/cfg/base.cfg

```diff
@@ -620,6 +624,22 @@ migrate_proto = "tcp"
 # Remember to open ports on both source and destination
 migrate_port = 49152
 
+##### migration_template use only parameters
+
+# Whether to migrate vm back to src host, default to yes 
+# migrate_vm_back = "yes"
+
+# virsh migrate options, default to "--live --p2p"
+# virsh_migrate_options = '--live --p2p'
+
+# Protocol used in migrate desturi, default to tcp
+# remote_protocol = 'tcp'
+
+# Migration thread timeout, default to 900s
+# thread_timeout = "900"
```

Comment:
rename thread_timeout to migrate_thread_timeout?

Reply:
Updated.